### PR TITLE
Final LS tests and release prep

### PR DESCRIPTION
**Pull request by jessemapel**
This PR adds the second LS test case and imaging locus tests.

It also sets the plugin release date for this Friday.

### CODE REVIEW

**Comment by jessemapel** [comment]
Closes #120 Also, once this is merged, we can merge to master and release.

File: tests/data/constAngularVelocityLineScan.json [hunk 1]
```diff
@@ -54,15 +54,15 @@
       [1000.0, 0.0, 0.0]
     ],
     "velocities": [
-      [0.0, 0.0, 0.0],
-      [0.0, 0.0, 0.0],
-      [0.0, 0.0, 0.0],
-      [0.0, 0.0, 0.0],
```
Comment (acpaquette) [inline tests/data/constAngularVelocityLineScan.json]: Any particular reason these got changed to 10s?

Comment (jessemapel) [reply tests/data/constAngularVelocityLineScan.json]: See #173 
The velocity isn't actually used besides defining cross-track and down-track. So, the velocity doesn't change the intersections or anything else.

File: tests/LineScanCameraTests.cpp [hunk 2]
```diff
@@ -46,87 +46,62 @@ TEST_F(ConstVelocityLineScanSensorModel, Inversion) {
...
+TEST_F(ConstVelocityLineScanSensorModel, ProximateImageLocus) {
+   csm::ImageCoord imagePt(8.5, 8.0);
+   csm::EcefCoord groundPt(5, 5, 5);
+   csm::EcefLocus locus = sensorModel->imageToProximateImagingLocus(imagePt, groundPt);
+   EXPECT_DOUBLE_EQ(locus.direction.x, -1.0);
+   EXPECT_DOUBLE_EQ(locus.direction.y,  0.0);
+   EXPECT_DOUBLE_EQ(locus.direction.z,  0.0);
+   EXPECT_DOUBLE_EQ(locus.point.x,      5.0);
+   EXPECT_DOUBLE_EQ(locus.point.y,      0.0);
+   EXPECT_DOUBLE_EQ(locus.point.z,      0.0);
```
Comment (jessemapel) [inline tests/LineScanCameraTests.cpp]: This point is the point on the imaging ray closest to the input ground point (AKA the projection of the ground point on the image ray).

Comment (jessemapel) [reply tests/LineScanCameraTests.cpp]: The direction is just the direction of the image ray

File: tests/LineScanCameraTests.cpp [hunk 3]
```diff
@@ -46,87 +46,62 @@ TEST_F(ConstVelocityLineScanSensorModel, Inversion) {
...
-   // EXPECT_NEAR(groundPt.z, -14.99325304, 1e-8);
+TEST_F(ConstVelocityLineScanSensorModel, RemoteImageLocus) {
+   csm::ImageCoord imagePt(8.5, 8.0);
+   csm::EcefLocus locus = sensorModel->imageToRemoteImagingLocus(imagePt);
+   EXPECT_DOUBLE_EQ(locus.direction.x, -1.0);
+   EXPECT_DOUBLE_EQ(locus.direction.y,  0.0);
+   EXPECT_DOUBLE_EQ(locus.direction.z,  0.0);
+   EXPECT_DOUBLE_EQ(locus.point.x,     10.0);
+   EXPECT_DOUBLE_EQ(locus.point.y,      0.0);
+   EXPECT_DOUBLE_EQ(locus.point.z,      0.0);
```
Comment (jessemapel) [inline tests/LineScanCameraTests.cpp]: This point is the ground point where the image ray intersects the body.

Comment (jessemapel) [reply tests/LineScanCameraTests.cpp]: The direction is just the direction of the image ray

File: tests/LineScanCameraTests.cpp [hunk 4]
```diff
@@ -46,87 +46,62 @@ TEST_F(ConstVelocityLineScanSensorModel, Inversion) {
...
-}
 
-TEST_F(ConstVelocityLineScanSensorModel, OffBody3) {
-   csm::ImageCoord imagePt(12.5, 4.0);
-   csm::EcefCoord groundPt = sensorModel->imageToGround(imagePt, 0.0);
-   // EXPECT_NEAR(groundPt.x, 0.44979759, 1e-8);
-   // EXPECT_NEAR(groundPt.y, 14.99325304, 1e-8);
-   // EXPECT_NEAR(groundPt.z, 14.99325304, 1e-8);
+TEST_F(ConstVelocityLineScanSensorModel, ProximateImageLocus) {
+   csm::ImageCoord imagePt(8.5, 8.0);
```
Comment (jessemapel) [inline tests/LineScanCameraTests.cpp]: For this test case and this image point, the image ray is f(t) = (1000, 0, 0) + t*(-1, 0, 0)

**Pull request merged**